llvm/test/Transforms/PhaseOrdering/X86/pr48844-br-to-switch-vectorization.ll

```diff
@@ -11,21 +11,71 @@ define dso_local void @test(ptr %start, ptr %end) #0 {
 ; CHECK-LABEL: @test(
```

Comment:
Time to replace above FIXME by a comment that under -O2 adjacent branches get fused into a switch statement before vectorization.